pres-fr/workshop08-pres-fr.Rmd

```diff
@@ -154,8 +154,50 @@ En revanche, les **GAM** font cela en ajustant une **fonction de lissage non-lin
 class: inverse, center, middle
 
 ## 2. Introduction aux GAMs
+---
+# Generalized Additive Models (GAM)
+
+Examinons un exemple! Nous allons utilsé le jeu de données `ISIT`.
+
+```{r, eval = FALSE, echo = TRUE}
+isit <- read.csv("data/ISIT.csv",sep=';')
+head(isit)
+```
+
+Ce jeu de donnée comporte des mesures de bioluminescence en relation � la profondeur (*depth*), la station de rechercher et la saison (*Season*).
```

Comment:
é character problem here too

Reply:
Yes I will, this is a problem with my french computer, the accent doesn't show in R (they show in this weird characters. I will try to fix it.